infrastructure/server/fam_admin_management_api.tf

```diff
@@ -0,0 +1,138 @@
+# Looking up a few things so they can be changed for this file in one place only
+
+data "aws_secretsmanager_secret" "db_admin_management_api_creds_secret" {
+  name = aws_secretsmanager_secret.famdb_admin_management_apicreds_secret.name
+}
```

Comment:
This `data` seems to be the same as the new one in flyway.tf, but it is ok, we don't have a common place to store these `data`, can refactor this later.

Reply:
oh yeah, I just followed the current way and added it. Think later on we can review these and think about how we want to define data variable.